app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,10 @@
     </div>
   <% end %>
   <div id="page-content-inner">
+    <% if @hero_html %>
+      <%= @hero_html.html_safe %>
```

Comment:
Just double checking that `hero_html` is not created from user input?

Reply:
@hero_html is from -> HtmlVariant.relevant.select(:html).find_by(group: "campaign", name: SiteConfig.campaign_hero_html_variant_name)&.html. The html for campaigns is uploaded by a very specific admin permission (not only a superuser) via internal, it's not open to the public.

Reply:
Thanks, just clarifying that we're not opening ourselves up to some exploit here.

Reply:
💯Thanks @citizen428